endaq/device/schemata/command-response.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Schema type="mide.ss.cmd" version="1" readversion="1">
+<Schema type="mide.ss.cmd" version="2" readversion="1">
```

Comment:
We've run into issues in the past by updating the version, will that happen again here?
I forget the exact issue, I think it was changing the IDE schema for the GPS

Reply:
The schema version change problem was an `idelib` thing. `ebmlite` itself doesn't care.